### PR TITLE
Tooltip for boosting depends on toot visibility (#1394)

### PR DESCRIPTION
**Pull request by expenses**
Fix for #1394. Looks like:
example
Note that this adds a new localization string.

### CODE REVIEW

**Comment by thurloat** [comment]
There's an inconsistency between your default translation, and the English translation.

`app/assets/javascripts/components/locales/en.jsx` calls it a Toot, though there are other translations in the file that refer to toots as Posts. i.e. `"compose_form.privacy_disclaimer"`.

`app/assets/javascripts/components/components/status_action_bar.jsx` calls it a Post.

**Comment by expenses** [comment]
@thurloat I think they're meant to be different.

**Review by yiskah — APPROVED** [review]
This looks to add a useful a11y feature

**Comment by Gargron** [comment]
@expenses @thurloat For the record, the official UI does not refer to any items as "toots". They are "statuses" in most places, and "posts" in places where it needs to be shorter. Only the "publish" button is called "toot". This is deliberate, the UI can't get *too* silly.

File: app/assets/javascripts/components/locales/en.jsx [hunk 1]
```diff
@@ -119,6 +119,7 @@ const en = {
   "status.mention": "Mention @{name}",
   "status.open": "Expand this status",
   "status.reblog": "Boost",
+  "status.cannot_reblog": "This toot cannot be boosted",
```
Comment (Gargron) [inline app/assets/javascripts/components/locales/en.jsx]: Change this to status and this is good to go. Sorry it took so long

**Review by Gargron — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/components/locales/en.jsx (hunk 1 above)
Comment (expenses) [inline app/assets/javascripts/components/locales/en.jsx]: .. And done!

**Review by Gargron — APPROVED** [review]

**Pull request merged**